Source/Core/VideoBackends/D3D/D3DState.h

```diff
@@ -48,6 +48,15 @@ union SamplerState {
   u64 packed;
 };
 
+union DepthState {
```

Comment:
Meh, maybe another quirk in clang-format, just like the other day with the else if condition being broken over two lines...fine by me if clang-format complains otherwise.